VideoWeb/VideoWeb.Common/Caching/DistributedConsultationInvitationCache.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Caching.Distributed;
+using Newtonsoft.Json;
+using VideoWeb.Common.Models;
+
+namespace VideoWeb.Common.Caching
+{
+    public class DistributedConsultationInvitationCache : IConsultationInvitationCache
+    {
+        private readonly IDistributedCache _distributedCache;
+
+        public DistributedConsultationInvitationCache(IDistributedCache distributedCache)
+        {
+            _distributedCache = distributedCache;
+        }
+
+        public async Task Write(ConsultationInvitation consultationInvitation)
+        {
+            var serialisedConference = JsonConvert.SerializeObject(consultationInvitation, CachingHelper.SerializerSettings);
+            var data = Encoding.UTF8.GetBytes(serialisedConference);
+            await _distributedCache.SetAsync(consultationInvitation.InvitationId.ToString(), data,
+                new DistributedCacheEntryOptions
+                {
+                    SlidingExpiration = TimeSpan.FromSeconds(2.5 * 60) // 2.5 minutes
```

Comment:
Does the exact same thing without the need to comment `TimeSpan.FromMinutes(2.5)`